mesos-commons/src/main/java/org/apache/mesos/collections/Maps.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.mesos.collections;
+
+import com.google.common.base.Predicate;
+
+import javax.annotation.Nonnull;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+/**
```

Comment:
I'll go with that... I was mirroring what guava does but I like your suggestion and I like that we can then use the import.